easybuild/easyblocks/o/openfoam.py

```diff
@@ -282,15 +307,16 @@ def sanity_check_step(self):
                                                     "refineMesh", "vorticity"]]
         # check for the Pstream and scotchDecomp libraries, there must be a dummy one and an mpi one
         if 'extend' in self.name.lower():
-            libs = [os.path.join(libsdir, x, "libPstream.so") for x in ["dummy", "mpi"]] + \
-                   [os.path.join(libsdir, "libscotchDecomp.so")]
+            libs = [os.path.join(libsdir, "libscotchDecomp.so")]
+            if LooseVersion(self.version) < LooseVersion('3.2'):
+                libs.extend([os.path.join(libsdir, x, "libPstream.so") for x in ["dummy", "mpi"]])
```

Comment:
I can't find anything on Pstream disappearing, and since this check is relatively recent (and not having the libPstream libraries went undetected for a while), cfr. https://github.com/hpcugent/easybuild-easyblocks/pull/436/files#diff-0edc74d66adc49c07871a687be0ef167R212, I'd like to get confirmation of this somehow...

I don't see any mention of Pstream in the OpenFOAM-Extend 3.2 release notes (http://sourceforge.net/p/openfoam-extend/foam-extend-3.1/ci/v3.2/tree/ReleaseNotes.txt), so it's fishy to me that this just disappeared...

Maybe @besserox has an idea?